### DISCUSSION TITLE
How to deploy next.js application on ibm cloud ?

### DISCUSSION THREAD

**Discussion by lakshmajee** [vercel/next.js#32994] [Help]
I have a next.js application and I wanted to deploy it on IBM cloud.
Could you guys suggest services in the IBM cloud?

Note: I knew vercel offers better solutions to this like managing and deploying a next.js application, but I am restricted to IBM cloud provider due to client requirements.

**Comment by sankalpmukim** [comment]
Come let's make an [Open Next](https://opennext.js.org/) adapter for IBM Cloud. 💪
